### DISCUSSION TITLE
Getting rid of Push To GitHub feature

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-starter#2262] [General]
Does anyone here use the "Push To GitHub" feature in https://launch.micronaut.io? It seems to be [broken](https://github.com/micronaut-projects/micronaut-starter/issues/2261), and instead of fixing it, I would prefer to get rid of it. 

The feature is not appealing. You have to grant permission to read/write and update Github actions for your public repositories for just one-time repository creation and push.  

![CleanShot 2024-01-02 at 14 44 09@2x](https://github.com/micronaut-projects/micronaut-starter/assets/864788/767805ab-5afe-49b5-a3fb-e703170c8b36)

All for the ability to push code to a GitHub repository, which our audience knows how to do or they should learn how to do. 

What do you think? Can we get rid of it? Or Should we keep it?

ping @micronaut-projects/core-developers

**Comment by wetted** [comment]
I think I've used the "push to github" feature once. 

I think we can remove the feature. It's easy enough to do it manually after the fact.

**Comment by timyates** [comment]
I've never used it, I hate giving anything access to my github

**Comment by timyates** [comment]
If we decide to remove it, I pushed a PR to do so in the UI (we can swing round to remove it from the back-end later)

https://github.com/micronaut-projects/micronaut-starter-ui/pull/108

**Reply by sdelamo** [reply]
We need to remove the code from starter as well.
